src/App.tsx

```diff
@@ -15,36 +15,119 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType;
+  isReversed: boolean;
+  isSorted: boolean;
```

Comment:
you never use isSorted so delete it everywhere ;)

Reply:
Ahh, ok :)